src/lib.rs

```diff
@@ -220,6 +226,15 @@ impl Stream for Incoming {
     }
 }
 
+impl Drop for Incoming {
```

Comment:
should probably be only implemented for unix sockets

Reply:
done